packages/gatsby-theme-newrelic/gatsby-browser.js

```diff
@@ -17,5 +18,25 @@ export const wrapRootElement = ({ element }, pluginOptions) => {
   );
 };
 
+export const onClientEntry = (_, { newrelic, i18n }) => {
```

Comment:
Out of curiosity, why did you choose `onClientEntry`? No wrong answer, just curious 😄 

[nit] To follow the pattern of the other hooks, could you also extract this to a `gatsby/on-client-entry.js` file? The only reason `wrapRootElement` is in here is because it differs in implementation from the `wrapRootElement` defined in `gatsby-ssr.js`

Reply:
Yes I could follow this pattern! and I chose `onClientEntry` it was the closest method to `window.onload` in the gatsby lifecycles hook. I could have also just done it all in `onRouterUpdate` (looking into it I really couldn't figure out a difference between those two other than the location properties that get passed in), so I could refactor this into `onRouterUpdate`.